activemq-kahadb-store/src/main/java/org/apache/activemq/store/kahadb/disk/page/Transaction.java

```diff
@@ -682,8 +681,8 @@ public void commit() throws IOException {
     public void rollback() throws IOException {
         if( writeTransactionId!=-1 ) {
             if (tmpFile != null) {
+                getTempFile().delete();
```

Comment:
Shouldn't this temp file delete come after the close of the RandomAccessFile?

Reply:
Good catch @cshannon  :D Thanks!

I just updated the PR :)